x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx

```diff
@@ -121,6 +120,17 @@ export const createAppRootMockRenderer = (): AppContextTestRender => {
     middlewareSpy.actionSpyMiddleware,
   ]);
 
```

Comment:
Note: moving this inside the mocked app context constructor. I put it outside by mistake last time.